.github/workflows/sanitizer_cuda.yml

```diff
@@ -3,20 +3,30 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'modules/cuda_plugin/**.*'
   workflow_dispatch:
 
 jobs:
   cuda-compute-sanitizer:
     runs-on: lohika-ci
-    timeout-minutes: 60
+    timeout-minutes: 300
     steps:
-    - name: fetch
-      run: git -C ~/runner/openvino_contrib fetch
-    - name: checkout
-      run: git -C ~/runner/openvino_contrib checkout $GITHUB_SHA
-    - name: CMake
+    - name: fetch openvino master branch
+      run: git -C ~/runner/openvino pull origin master
+    - name: checkout openvino submodules
+      run: git -C ~/runner/openvino submodule update --init --recursive
+    - name: checkout openvino_contrib master branch
+      run: git -C ~/runner/openvino_contrib checkout master
+    - name: delete openvino_contrib temporary branch
+      run: git -C ~/runner/openvino_contrib branch -D temporary-ci-branch | echo 1
+    - name: fetch openvino_contrib
+      run: git -C ~/runner/openvino_contrib fetch origin $GITHUB_REF:temporary-ci-branch
+    - name: checkout openvino_contrib
+      run: git -C ~/runner/openvino_contrib checkout temporary-ci-branch
+    - name: cmake
       run: env -C ~/runner ./configure_cuda_sanitizer.sh
     - name: ninja
       run: echo 'all:; +NINJA_STATUS="[%u/%r/%f] " ninja -C ~/runner/build_cuda_sanitizer -v CudaFuncTests CudaUnitTests' | make -j`nproc` -f-
     - name: functional tests
-      run: env -C ~/runner PATH=/usr/local/cuda-11.2/bin:$PATH ./openvino_contrib/modules/cuda_plugin/utils/cuda-sanitizer.sh ./build_cuda_sanitizer/bin/intel64/CudaFuncTests --gtest_filter=*smoke*:-*dynamic*:*Dynamic*:smoke_GRU*:smoke_LSTM*:smoke_TensorIterator*:*ConvBiasFusion*
+      run: env -C ~/runner PATH=/usr/local/cuda-11.2/bin:/usr/local/cuda-11.4/bin:/usr/local/cuda-11.7/bin:$PATH ./openvino_contrib/modules/cuda_plugin/utils/cuda-sanitizer.sh ./build_cuda_sanitizer/bin/intel64/CudaFuncTests --gtest_filter=*smoke*:-*dynamic*:*Dynamic*:smoke_GRU*:smoke_LSTM*:smoke_TensorIterator*:*ConvBiasFusion*:*smoke*OVExecGraphImportExportTest.importExportedIENetwork*:*smoke*OVClassBasicTestP.registerNewPluginNoThrows*:*smoke*OVHoldersTest.Orders*:*smoke*IEClassBasicTestP.registerNewPluginNoThrows*:*smoke*IEClassBasicTestP.smoke_registerPluginsXMLUnicodePath*
```

Comment:
Could you comment on why those tests need to be excluded?

Reply:
They was excluded, because under cuda-memcheck a memory usage increased a lot and these tests lead to crash due to not enough memory. It happens only under cuda-memcheck.

Except these tests: `*smoke*OVHoldersTest.Orders*:*smoke*IEClassBasicTestP.registerNewPluginNoThrows*:*smoke*IEClassBasicTestP.smoke_registerPluginsXMLUnicodePath*`,
them I disable because these tests underhood search shared library near executable binary, but in case of cuda-memcheck, executable binary is cuda-memcheck and due to this a library could not be found. For now I decided that these 2 tests is not very important for cuda-memcheck and fix it would be very harder then benefit of these 2 tests under cuda-memcheck

Reply:
Ok Please copy this explanation into commit message

Reply:
I've already add explanation in commit:
_Updated history_cuda.yml, sanitizer_cuda.yml and test_cuda.yml pipelines:
1) All cuda_plugin pipelines will be run if cuda_plugin fiolder was changed
**2) Updated sanitizer_cuda.yml with proper CudaFuncTests's filters due to memory exhausting under cuda-memcheck**_

Do you want to extend it ?